libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/infopanel/InfoPanelRoutePreviewButtonBinder.kt

```diff
@@ -4,20 +4,23 @@ import android.view.ViewGroup
 import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
 import com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver
 import com.mapbox.navigation.dropin.extendablebutton.RoutePreviewButtonComponent
+import com.mapbox.navigation.dropin.internal.extensions.reloadOnChange
 import com.mapbox.navigation.dropin.navigationview.NavigationViewContext
 import com.mapbox.navigation.ui.base.lifecycle.UIBinder
+import com.mapbox.navigation.ui.base.view.MapboxExtendableButton
 
 @ExperimentalPreviewMapboxNavigationAPI
 internal class InfoPanelRoutePreviewButtonBinder(
     private val context: NavigationViewContext
 ) : UIBinder {
 
     override fun bind(viewGroup: ViewGroup): MapboxNavigationObserver {
-        return RoutePreviewButtonComponent(
-            context.store,
-            viewGroup,
-            context.styles.routePreviewButtonParams,
-            context.routeOptionsProvider,
-        )
+        return reloadOnChange(context.styles.routePreviewButtonStyle) { style ->
+            val button = MapboxExtendableButton(viewGroup.context, null, 0, style)
+            viewGroup.removeAllViews()
+            viewGroup.addView(button)
+
+            RoutePreviewButtonComponent(context.store, context.routeOptionsProvider, button)
```

Comment:
Similar to what we are doing for `InfoPanelEndNavigationButtonBinder`
```
button.updateMargins(
                right = button.resources.getDimensionPixelSize(R.dimen.mapbox_infoPanel_paddingEnd)
```
I see we are not doing for `InfoPanelRoutePreviewButtonBinder` and `InfoPanelStartNavigationButtonBinder`. Looking at it in details I see that for 
- DestinationPreview: Contains `RoutePreview` and `StartNavigation` buttons
- RoutePreview: Contains `StartNavigation` buttons,
the margins are assigned in the XML. We should be consistent and remove the margins from the XML. Let us add the margins in code here. Same applies to `InfoPanelStartNavigationButtonBinder`

Reply:
Ok. Let me see what I can do.

Reply:
Done. Please see updated version.